modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators.Tests/TestData/Sources/ExportDiagnostics_GD0107.cs

```diff
@@ -1,19 +1,68 @@
 using Godot;
+using Godot.Collections;
 
 public partial class ExportDiagnostics_GD0107_OK : Node
 {
     [Export]
     public Node NodeField;
 
+    [Export]
+    public Node[] DotnetArrayOfNodesField;
```

Comment:
nit: We usually use the word `System` here.

```suggestion
    public Node[] SystemArrayOfNodesField;
```